docs/style/color.md

```diff
@@ -1,22 +1,173 @@
-| ![](../.gitbook/assets/color_independence-cranberry.png) | `#680A1D` | **INDEPENDENCE CRANBERRY** | This color is used as an accent color, replacing the Berkshires Green, on the G2G site. It also replaces the Mass Blue in some areas on the Service page template. Examples:  None \(Deprecated\) |
-| ![](../.gitbook/assets/color_independence-cranberry-50.png) | `#AD7E88` | **50% INDEPENDENCE CRANBERRY** | Examples:  Heading underline/Navigation active state underline |
-| ![](../.gitbook/assets/color_error-red%20%281%29.png) | `#CD0D0D` | **ERROR RED** | This color is used for error language, borders, and icons. Examples:  Form error states |
-| ![](../.gitbook/assets/color_error-red.png) | `#FCF3F3` | **5% ERROR RED** | Examples:  Overall form error state background |
+Where appropriate, we enable people to introduce their own color palettes. Our products adapt intelligently and flexibly to cater to people's preferences. We are also committed to complying with AA standard contrast ratios. To do this, choose primary, secondary, and extended colors that support usability by ensuring sufficient color contrast between elements so that people with low vision can see and use the interface.
+
+### Primary color palette
+
+Our primary palette is comprised of neutrals, blue, and green. These colors are present across most touch points from marketing to product. 
+
+## Blue
+
+### Bay Blue
+
+![HEX:  \#14558F  RGB:  20,85,143](../.gitbook/assets/color_bay-blue_long%20%281%29.png)
+
+This complementary color is used for major headings, underlines on rich text headings, callouts, icons, and as a background accent color.
+
+* Links
+* Buttons
+* Glorified buttons
+* Service page header overlay
+* Relationship indicator borders
+* Pagination
+* Jump links
+
+### Bay Blue \(50%\)
+
+![HEX:  \#8AAAC7  RGB:  138,170,199](../.gitbook/assets/color_bay-blue_50_long.png)
+
+* Illustrated card border color/Button border color/File type icons/Link arrows
+
+### Bay Blue \(10%\)
+
+![HEX:  \#E7EEF4  RGB:  231,238,244](../.gitbook/assets/color_bay-blue_10_long.png)
+
+* Glorified button fill
+
+## Green
+
+### Berkshires green
+
+![HEX:  \#43956F  RGB:  67,149,111](../.gitbook/assets/color_berkshires-green_long.png)
+
+This complementary color is used for major headings, underlines on rich text headings, callouts, icons, and as a background accent color.
+
+* ‘In this section’ table of contents heading background color
+* ‘The details’ sticky left navigation heading background color
+* Icon background colors \(topic pages, footer icons, decorative bullet points\)
+* Icon color fill \(search bar, location icons, etc.\)
+* Page dividers
+
+### Berkshires green \(50%\)
+
+![HEX:  \#A1CAB7  RGB:  161,202,183](../.gitbook/assets/color_berkshires-green_50_long.png)
+
+* **Heading underline**
+* **Heading underlines \(primary nav active state\)**
+
+### Berkshires green \(10%\)
+
+![HEX:  \#D9EAE2  RGB:  217, 234, 226](../.gitbook/assets/color_berkshires-green_10_long.png)
+
```

Comment:
Berkshires green 10% seems to be missing descriptions

Reply:
I would recommend to bold the color overview section and unbold the examples for consistency. This list is not up to date, e.g. we can remove the G2G reference based on yesterday's conversation. Also there are a few colors currently missing an overview maybe this is an opportunity to add that context in. Otherwise LGTM!

Reply:
Good catch. Its actually supposed to be 20% too. I fixed that and the details. I updated the head content in H3 so it would not show as a content point in the right nav. Publishing changes now.